src/page.ts

```diff
@@ -13,7 +13,7 @@ export const createPageManager = (event: MCEvent, settings: TrackerSettings) =>
       manager.set('pvid', uuidv4());
       manager.set('email', getEventPayloadValue(event, 'customerEmail'));
     },
-    set: (key: PageVariable, value: string) => {
+    set: (key: PageVariable, value: string): void => {
       if (event.client.get(key)) {
         return;
       }
```

Comment:
the getter here (below) should also fallback to an empty string in case the requested variable is undefined